src/components/Homepage/subcomponents/CategoriesSection.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import styles from './Categories.module.sass'
+import categoryEarphones from '@/assets/categories-earphones.png'
+import categoryHeadphones from '@/assets/categories-headphones.png'
+import categorySpeakers from '@/assets/categories-speakers.png'
+import Image from 'next/image'
+import RightArrow from '@/svgs/RightArrow'
+const CATEGORIES_LIST = ['Headphones', 'Speakers', 'Earphones']
+
+const CategoriesSection = () => {
+  return (
+    <section className={styles.sectionCategories}>
+      {CATEGORIES_LIST.map((category) => (
+        <div className={styles.categoryCard} key={category}>
+          <div className={styles.imageContainer}>
+            <Image
+              src={
+                category === 'Headphones'
```

Comment:
you can refactor this to

```
type CategorySrc = {
  [key: string]: string;
};

const categorySrc: CategorySrc = {
  Headphones: categoryHeadphones.src,
  Speakers: categorySpeakers.src,
  Earphones: categoryEarphones.src,
};

const src = categorySrc[category];
```

Reply:
Refactored. This is much cleaner than my code.